jetstream/cli.py

```diff
@@ -326,6 +399,13 @@ def convert(self, value, param, ctx):
     help="Kubernetes cluster certificate used for authenticating to the cluster",
 )
 
+recreate_enrollments_option = click.option(
+    "--recreate_enrollments",
+    "--recreate-enrollments",
+    help="Recreate the enrollments tables",
+    default=False,
+)
+
```

Comment:
`--recreate-enrollments=True` when rerunning the analysis. Todo: update jetstream-config CI config